engine/src/main/java/com/redhat/cloud/notifications/processors/InsightsUrlsBuilder.java

```diff
@@ -105,7 +106,15 @@ public String buildApplicationUrl(JsonObject data, String integration_type) {
             // Settings paths
             case "integrations", "notifications" -> path + "settings/" + application;
             // OpenShift path override
-            case "cluster-manager" -> path;
+            case "cluster-manager" -> {
```

Comment:
Since we are no longer relying on a single common template for chat notifications, it should be great to have some templates dedicated for OCM with their own logic to build links.

For the current requirement, we can apply the same model for all their event types.

Reply:
Changes implemented with a new `OcmClusterUrlExtension` in common-template. If the environment_url is not found within `data.context`, it will fallback to using the `application_url`.